Documentation/ApiOverview/CommandControllers/Index.rst

```diff
@@ -17,6 +17,14 @@ These commands can also be run from the TYPO3
 :ref:`scheduler <symfony-console-commands-scheduler>` if this option is not
 disabled in the :file:`Configuration/Services.yaml`.
 
+The starting point for the commands differs depending on the kind of your TYPO3 installation.  
+
+ 1. In installations with composer TYPO3 the starting point is usually the folder where the
```

Comment:
This sentence is hard to understand. Additionally, it is at least misunderstanding: "starting point the folder where the file composer.json" -> there are many composer.json's in a typical TYPO3 installation. And no one wants to add var and vendor in a folder of an extension.